packs/pfs-season-2-bestiary/perizia-5-6.json

```diff
@@ -1452,7 +1452,24 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "damage-roll",
+                        "key": "RollOption",
+                        "option": "skewer",
+                        "toggleable": true
+                    },
+                    {
+                        "damageType": "bleed",
```

Comment:
Not sure on how strict PFS is on critical bleed being 2d4 vs (2*1d4).

Reply:
The system should be using whatever setting the GM has chosen.